--- mariadb/query14.sql	2026-09-23 19:55:01
+++ /tmp/autoadapt/query14.sql	2026-09-23 19:55:01
@@ -30,7 +30,7 @@
      ,date_dim d3
  where ws_item_sk = iws.i_item_sk
    and ws_sold_date_sk = d3.d_date_sk
-   and d3.d_year between 1998 AND 1998 + 2)
+   and d3.d_year between 1998 AND 1998 + 2) as TBL_1
  where i_brand_id = brand_id
       and i_class_id = class_id
       and i_category_id = category_id
