--- mariadb/query23.sql	2026-09-23 19:55:01
+++ /tmp/autoadapt/query23.sql	2026-09-23 19:55:01
@@ -17,7 +17,7 @@
         where ss_customer_sk = c_customer_sk
          and ss_sold_date_sk = d_date_sk
          and d_year in (1999,1999+1,1999+2,1999+3) 
-        group by c_customer_sk)),
+        group by c_customer_sk) as TBL_1),
  best_ss_customer as
  (select c_customer_sk,sum(ss_quantity*ss_sales_price) ssales
   from store_sales
@@ -45,7 +45,7 @@
          and d_moy = 1 
          and ws_sold_date_sk = d_date_sk 
          and ws_item_sk in (select item_sk from frequent_ss_items)
-         and ws_bill_customer_sk in (select c_customer_sk from best_ss_customer)) 
+         and ws_bill_customer_sk in (select c_customer_sk from best_ss_customer)) as TBL_2 
  limit 100;
 
 with frequent_ss_items as
@@ -67,7 +67,7 @@
         where ss_customer_sk = c_customer_sk
          and ss_sold_date_sk = d_date_sk
          and d_year in (1999,1999+1,1999+2,1999+3)
-        group by c_customer_sk)),
+        group by c_customer_sk) as TBL_3 ),
  best_ss_customer as
  (select c_customer_sk,sum(ss_quantity*ss_sales_price) ssales
   from store_sales
@@ -100,6 +100,6 @@
          and ws_item_sk in (select item_sk from frequent_ss_items)
          and ws_bill_customer_sk in (select c_customer_sk from best_ss_customer)
          and ws_bill_customer_sk = c_customer_sk
-       group by c_last_name,c_first_name) 
+       group by c_last_name,c_first_name) as TBL_4 
      order by c_last_name,c_first_name,sales
   limit 100;
