--- mariadb/query2.sql	2026-09-23 19:55:01
+++ /tmp/autoadapt/query2.sql	2026-09-23 19:55:01
@@ -7,7 +7,7 @@
         union all
         select cs_sold_date_sk sold_date_sk
               ,cs_ext_sales_price sales_price
-        from catalog_sales)),
+        from catalog_sales) as TBL_1),
  wswscs as 
  (select d_week_seq,
         sum(case when (d_day_name='Sunday') then sales_price else null end) sun_sales,
